lib/backends/sdc/containers.js

```diff
@@ -1735,6 +1953,21 @@ function buildVmPayload(opts, container, callback) {
             addNetworksToPayload(opts, container, payload, cb);
         },
 
+        function addExternalNet(_, cb) {
+            if (publishingPorts(container) || opts.fabricRequireExternal) {
+                // external must be primary.
+                payload.networks.forEach(function (nw) {
+                    delete nw.primary;
+                });
+
+                payload.networks.push(
+                    { uuid: opts.config.overlay.externalPool, primary: true });
```

Comment:
`opts.config.overlay.externalPool` will always be defined if fabrics are enabled, right?